fw/http.c

```diff
@@ -3839,6 +3882,12 @@ tfw_h2_adjust_req(TfwHttpReq *req)
 				continue;
 			}
 			break;
+		case TFW_HTTP_HDR_COOKIE:
+			if (TFW_STR_DUP(field)) {
+				r |= write_merged_cookie_headers(&ht->tbl[TFW_HTTP_HDR_COOKIE], &it);
+				continue;
+			}
+			break;
```

Comment:
```suggestion
			if (!TFW_STR_DUP(field))
			        break;
			r |= write_merged_cookie_headers(&ht->tbl[TFW_HTTP_HDR_COOKIE], &it);
			continue;
```

Reply:
The good point about line width limit is that it also requires to keep the code with smaller nesting (e.g. see also Code Complete book).